src/cargo/ops/cargo_test.rs

```diff
@@ -212,6 +214,13 @@ fn run_doc_tests(
         gctx.shell().status("Doc-tests", unit.target.name())?;
         let mut p = compilation.rustdoc_process(unit, *script_meta)?;
 
+        let color_arg = match color {
+            ColorChoice::Always => "always",
+            ColorChoice::Never => "never",
+            ColorChoice::CargoAuto => "auto",
+        };
+        p.arg("--color").arg(color_arg);
+
```

Comment:
Mind moving this later? Within this function, we had "add env, add args" and now we have "add arg, add env, add args"

Also, as indicated by the test suite, some tests will need to be updated that are checking for some but not all arguments